crates/pixi_build_frontend/src/tool/mod.rs

```diff
@@ -14,13 +14,13 @@ pub use installer::ToolContext;
 /// A tool that can be invoked.
 #[derive(Debug)]
 pub enum Tool {
-    Isolated(Arc<IsolatedTool>),
+    Isolated(IsolatedTool),
     System(SystemTool),
     Io(InProcessBackend),
 }
 
 impl Tool {
-    pub fn as_isolated(&self) -> Option<Arc<IsolatedTool>> {
+    pub fn as_isolated(&self) -> Option<IsolatedTool> {
```

Comment:
`as_*` implies that you return the inner thing as a reference. `Option<&IsolatedTool>`.